src/EditorFeatures/CSharpTest/Completion/CompletionProviders/XmlDocumentationCommentCompletionProviderTests.cs

```diff
@@ -1015,23 +1044,6 @@ class C
             await VerifyItemExistsAsync(text, "langword");
         }
 
-        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/11489")]
-        public async Task AttributeValueOnQuote()
-        {
-            var text = """
-                class C
-                {
-                    /// <summary>
-                    /// <see langword="$$
-                    /// </summary>
-                    static void Goo()
-                    {
-                    }
-                }
-                """;
-            await VerifyItemExistsAsync(text, "await", usePreviousCharAsTrigger: true);
-        }
```

Comment:
why was this test removed? generally, we never do that.  we just update the test to show the new behavior.

Reply:
Oh I thought you wanted me to move it instead of duplicating it, I'll add it back

Reply:
nope!  please keep :)